src/sentry/sentry_metrics/indexer/limiters/writes.py

```diff
@@ -162,7 +171,11 @@ def check_write_limits(
 
         state = RateLimitState(
             _writes_limiter=self,
-            _use_case_id=use_case_id,
+            _use_case_id=(
+                UseCaseKey.RELEASE_HEALTH
+                if use_case_id is UseCaseID.SESSIONS
+                else UseCaseKey.PERFORMANCE
+            ),
```

Comment:
Is there no place where we're mapping `use_case_id`s to `UseCaseKey` prior to this PR? If that mapping needs to exist then I think it should be in one central location and this file shouldn't contain the logic (e.g. this file should call  `get_use_case_key_from_use_case_id(use_case_id)` and not be coupled to how that works).

I think this also would clean up some tests below

Reply:
Yep, we do, it's in `use_case_id_registry.py`

Reply:
I instead of a function I added a map of `UseCaseKey` -> `UseCaseID` because it should be a hard error
```python
METRIC_ID_MAPPING: Mapping[UseCaseKey, UseCaseID] = {
    UseCaseKey.RELEASE_HEALTH: UseCaseID.SESSIONS,
    UseCaseKey.PERFORMANCE: UseCaseID.TRANSACTIONS,
}
```